src/open_inwoner/templates/pages/cases/status.html

```diff
@@ -32,14 +32,23 @@ <h2 class="h2" id="documents">{% trans 'Documenten' %}</h2>
                 {% endif %}
 
                 {% if case.internal_upload_enabled %}
-                    <h2 class="h2" id=>{% trans "Document toevoegen" %}</h2>
+                    <h2 class="h2">{% trans "Document toevoegen" %}</h2>
+                    {% if case.case_type_config_description %}
+                        <p class="p">{{case.case_type_config_description}}</p>
+                    {% endif %}
+
                     <p class="p">
                         {% blocktranslate with max_filesize=openzaak_config.max_upload_size allowed_extensions=case.allowed_file_extensions|join:', ' %}
```

Comment:
As discussed with Groningen, if specified the case_type_config_description will overrule this default description.

So the idea is that we either show the default (with max filesize and the formats allowed) or show the case type description, but not both